docs/dev-environment.md

```diff
@@ -5,16 +5,17 @@ title: Setup Dev Environment
 
 ## Setup Backend
 
-Download go dependencies with [golang/dep](https://github.com/golang/dep).
+Download go dependencies with go modules enabled.
 
 ```
-$ dep ensure
+$ export GO111MODULE=on
```

Comment:
This should only be needed if the project is inside the gopath maybe this information could be added.
```suggestion
$ export GO111MODULE=on # only needed if the project is inside $GOPATH/src
```